style.css

```diff
@@ -0,0 +1,114 @@
+#bgimage {
+  position: absolute;
+  background-image: url("images/Illustration.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  padding-left: 165%;
+  padding-top: 900px;
+  z-index: 1;
+}
```

Comment:
Great job so far @StarMindz 🤝

- [ ] Just like the previous reviewer pointed out. Try to fix the white space on the right side of your design. You may try to remove the padding-left. It seems it's the value-creating this white space.
- Your design
![padding](https://user-images.githubusercontent.com/63917725/176956795-85677c89-1ada-4fcb-8d70-20f5ca8d316e.PNG)